util/MsSql/crontab

```diff
@@ -1,3 +1,3 @@
-0 0 * * * bitwarden /backup-db.sh >> /var/log/cron.log 2>&1
+50 23 * * * bitwarden /backup-db.sh >> /var/log/cron.log 2>&1
```

Comment:
Could we change this to `59 23 * * *`? Why the 10 minute gap?

Reply:
You're right, no real reason, done 👍